pkg/nodeps/values.go

```diff
@@ -207,7 +209,7 @@ func GetValidMariaDBVersions() []string {
 // IsValidPostgresVersion is a helper function to determine if a PostgreSQL version is valid, returning
 // true if the supplied version is valid and false otherwise.
 func IsValidPostgresVersion(v string) bool {
-	if _, ok := ValidPostgresVersions[v]; !ok {
+	if _, ok := ValidPostgresVersions[strings.TrimPrefix(v, Postgres+":")]; !ok {
```

Comment:
Should this be `Postgres` or `postgres` ?

Reply:
`Postgres`, this is a constant:

https://github.com/ddev/ddev/blob/195334dbf8ea974453183b535dc213c980edca63/pkg/nodeps/values.go#L17-L22